WLC/code_executor/haskell_executor.py

```diff
@@ -0,0 +1,16 @@
+from ..code_fixing.python_code_fixer import PythonCodeFixer
+from ..code_executor.abstract_executor import AbstractCodeExecutor
+
+
+class HaskellExecutor(AbstractCodeExecutor):
```

Comment:
Will this be implemented before the PR is accepted?

Reply:
Nope, this is just added the skeleton. That will be implemented soon.